content/en/docs/concepts/cluster-administration/flow-control.md

```diff
@@ -0,0 +1,312 @@
+---
+title: Manage API Server load with the Flow Control API
+content_template: templates/concept
+min-kubernetes-server-version: v1.18
+---
+
+{{% capture overview %}}
+
+Controlling the behavior of the Kubernetes API server in an overload situation
+is a key task for cluster administrators. The `kube-apiserver` has some controls
+available (i.e. the `--max-requests-inflight` and
+`--max-mutating-requests-inflight` command-line flags) to limit the amount of
+outstanding work that will be accepted, preventing a flood of inbound requests
+from overloading and potentially crashing the API server, but beyond that, the
+Priority and Fairness (or flow-control) API provides a configurable mechanism to
+ensure that in a high-load situation, forward progress will be made on the right
+requests. It also allows for a limited amount of queuing, so that in the event
+of _bursty_ traffic that remains below acceptable levels on average, no requests
+need be dropped.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Enabling the Flow Control API
+
+{{< feature-state state="alpha" >}}
+
+The Flow Control API is part of the API Priority and Fairness
+[feature](https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/20190228-priority-and-fairness.md),
+which is presently in alpha. To enable it, add the following command-line flags
```

Comment:
```suggestion
which is not enabled by default. To enable it, add the following command-line flags
```
because the feature-state shortcode already says that the feature is alpha.